syntax = "proto3";

package test.upstreams.http;

message CustomHeaderValidator {
}
